src/jit/codegencommon.cpp

```diff
@@ -10317,10 +10333,16 @@ void CodeGen::genFuncletProlog(BasicBlock* block)
 
     compiler->unwindBegProlog();
 
-    // TODO Save callee-saved registers
+    inst_RV(INS_push, REG_FPBASE, TYP_REF);
```

Comment:
@BruceForstall It seems that ``UnwindEbpDoubleAlignFrame`` should be revised along with ``genFuncletProlog``.